diff -ru wl-clip-persist-base/Cargo.toml wl-clip-persist/Cargo.toml
--- wl-clip-persist-base/Cargo.toml	2024-03-25 11:35:22.147732953 +0100
+++ wl-clip-persist/Cargo.toml	2024-03-25 11:35:51.099475419 +0100
@@ -15,6 +15,6 @@
 clap = { version = "3.2.8", features = ["cargo", "derive"] }
 filedescriptor = "0.8.2"
 libc = "0.2.126"
-fancy-regex = "0.10.0"
+fancy-regex = "0.11.0"
 wayland-protocols = { version = "0.29.4", features = ["client", "unstable_protocols"] }
 wayland-client = { version = "0.29.4", features = ["use_system_lib"] }
